types/loopback/index.d.ts

```diff
@@ -146,7 +147,7 @@ declare namespace l {
              * ``
              *  @returns {Array} Array of model classes
              */
-            models(): any[];
+            models(): Array<typeof Model>;
```

Comment:
Actualy not, because `models()` returns not an instance of `Model` but the class itself.